src/robusta/core/model/runner_config.py

```diff
@@ -29,6 +29,7 @@
 from robusta.model.alert_relabel_config import AlertRelabel
 from robusta.model.playbook_definition import PlaybookDefinition
 from robusta.utils.base64_utils import is_base64_encoded
+from src.robusta.core.sinks.sink_config import SinkConfigBase
```

Comment:
please change the import to `module` style, i.e.:
`robusta.code.sinks...`

Using the `src` directory causes issues when running it directly from the ide

Reply:
Done, thank you for spotting this.